loris/README.md

```diff
@@ -32,3 +32,34 @@ You can run Loris locally with the following two commands:
 where `$PLATFORM` is the root of the platform repo, and `$LORIS` is the root of a local checkout of the Loris repo.
 
 You'll then have a development version of Loris running at <http://localhost:8888>.
+
+## Testing Loris with nginx
+
+In order to test Loris with it's nginx config this project provides a `docker-compose.yml`
+
+This can be run using (substituting env variables as necessary):
+
+### docker-compose
+
+This starts both loris and nginx (at the tag specified), using the config file specified.
+
+```sh
+CONFIG_KEY=location/of/config.ini \
+NGINX_TAG=latest \
+AWS_ACCESS_KEY_ID=key_id \
+AWS_SECRET_ACCESS_KEY=access_key \
+docker-compose up
+
+```
+
+### curl localhost:9000
+
+You can then query the local version of loris (in order to avoid redirects it's necessary to use the `ELB-HealthChecker/2.0` User-Agent).
+
+```sh
+curl \
+    -A ELB-HealthChecker/2.0 \
+    -v \
+    -L \
+    http://localhost:9000/image/V0017087.jpg/full/300,/0/default.jpg > default.jpg
```

Comment:
minor: could we get the full names of these options spelt out? Most people will copy-paste this, so verbosity beats terseness.

e.g. `--user-agent "ELB-HealthChecker/2.0"`

Also, you might want to use `--output default.jpg` rather than redirecting stdout.

Reply:
👍 good point